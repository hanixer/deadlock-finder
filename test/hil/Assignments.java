public class Assignments {
    static void func1(int a, int b, int c, int d) {
        a += b;
        d -= c;
        int x = (a *= b) + b;
        int y = (b /= c);
    }
}