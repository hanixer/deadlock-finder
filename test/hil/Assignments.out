func Assignments.func1(a: int, b: int, c: int, d: int): void {
  a = a+b
  d = d-c
  a = a*b
  var x: int = a+b
  b = b/c
  var y: int = b
}